lib/utils/export/index.js

```diff
@@ -10,6 +10,9 @@ import {
 	uniqueId,
 } from 'lodash';
 
+export const CRLF_NEWLINE = '\r\n';
```

Comment:
`LF_ONLY_NEWLINES` I think is helpful. `CRLF_NEWLINE`'s value might be a little more dubious. might actually hamper readability since it's a straightforward constant. I support you either way, but I wouldn't have turned this one ^^^ into a named constant